File: Marvellous_LB/4-Pattern printing/program22.c
#include<stdio.h>

void Display(int iRows,int iCols)
{
    int iR=0,iC=0;

    if(iRows!=iCols)
    {
        printf("Can not print diagonal pattern...\n");
        return ;
    }

    for(iR=1;iR<=iRows;iR++)
    {
        for(iC=1;iC<=iCols;iC++)
        {
             if(iC==iR)
            {
                printf("#\t");
            }
            else
            {
                printf("%d\t",iR);
            }
        }
        printf("\n");
    }
}

int main()
{
    int iRows=0,iCols=0;

    printf("enter number of rows:\t");
    scanf("%d",&iRows);

    printf("enter number of columns:\t");
    scanf("%d",&iCols);

    Display(iRows,iCols);

    return 0;
}

File: Marvellous_LB/8-JAVA/program22.java
import java.util.*;

class Pattern
{
    public void Display(int iRows,int iCols)
    {
        int i=0,j=0;

        for(i=1;i<=iRows;i++)
        {
            for(j=1;j<=iCols;j++)
            {
                System.out.println("*\t");
            }
            System.out.println();
        }
    }
}

class program22
{
    public static void main(String a[])
    {
        int iRow=0,iCol=0;
        Pattern pobj=new Pattern();
        Scanner sobj= new Scanner(System.in);

        System.out.println("Enter number of rows : ");
        iRow=sobj.nextInt();

        System.out.println("nter number of columns : ");
        iCol=sobj.nextInt();

        pobj.Display(iRow,iCol);
    }
}

